data/core/editor/help.cfg

```diff
@@ -337,13 +382,19 @@ The <italic>text='map_file'</italic> method is preferred over using a preprocess
 
 The map data can stored as part of a scenario’s .cfg file, directly in the <italic>text='map_data'</italic> attribute. In other words, in the place that the preprocessor would include it when using the preprocessor-include method.
 
-<bold>text='Using Embedded Format in Terrain Mode'</bold>
+<header>text='Using Embedded Format in Terrain Mode'</header>
 
 If you are editing the map and not using the Scenario Mode support, then it’s trivial to move the data to a native map file before opening it in the editor. This conversion is recommended — the editor supports editing the content of map_data while leaving everything else in the file untouched, but this is rarely-used code. Maps opened this way are marked (E) in the Window menu.>> + "
```

Comment:
Found some reference documentation for this, https://github.com/wesnoth/wesnoth/blob/ff0eecc7c7e8e896e1e53439a103d3991fd97289/src/editor/map/map_context.cpp#L120-L139